nixos/tests/web-apps/mastodon/remote-postgresql.nix

```diff
@@ -69,18 +71,14 @@ in
             tryFiles = "$uri @proxy";
           };
           locations."@proxy" = {
-            proxyPass = "http://192.168.2.201:55001";
-            proxyWebsockets = true;
-          };
-          locations."/api/v1/streaming/" = {
```

Comment:
For anybody wondering: Yes this was removed without adequate replacement. The test still succeeds and in fact it was never testing the old streaming anyway, due to a wrong port number (55002 instead of 55000).